README.md

```diff
@@ -1,6 +1,6 @@
 # mdeval.nvim
 
-This plugin allows you evaluate code blocks inside markdown documents.
+This plugin allows you evaluate code blocks inside markdown,vimwiki and norg documents.
```

Comment:
I suggest to insert some links here, because some users may not be aware of these projects.

```suggestion
This plugin allows you evaluate code blocks inside markdown, [vimwiki](https://github.com/vimwiki/vimwiki) and [norg](https://github.com/vhyrro/neorg) documents.
```